.gitignore

```diff
@@ -127,7 +127,4 @@ Gemfile.local
 /public/packs-test
 /node_modules
 
-!/public/packs
```

Comment:
So, our production environment needs nodejs and npm of the correct version, right?

Reply:
Yes, so that will mean updating. Still need to solve circle ci, then communicate with @3scale/operations  and see how we could make it happen